lib/create_azure_ad_oauth_config.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023-2024 the Deno authors. All rights reserved. MIT license.
+import type { OAuth2ClientConfig } from "../deps.ts";
+import { getRequiredEnv } from "./get_required_env.ts";
+
+/**
+ * Returns the OAuth configuration for Azure.
+ *
+ * Requires `--allow-env[=AZURE_AD_CLIENT_ID,AZURE_AD_CLIENT_SECRET,AZURE_AD_TENANT_ID]`
+ * permissions and environment variables:
+ * 1. `AZURE_AD_CLIENT_ID`
+ * 2. `AZURE_AD_CLIENT_SECRET`
+ * 4. `AZURE_AD_TENANT_ID`
+ *
+ * @example
+ * ```ts
+ * import { createAzureADOAuthConfig } from "https://deno.land/x/deno_kv_oauth@$VERSION/mod.ts";
+ *
+ * const oauthConfig = createAzureADOAuthConfig({
+ *   redirectUri: "http://localhost:8000/callback",
+ *   scope: ["openid", Deno.env.get("AZURE_AD_CLIENT_ID")!]
+ * });
+ * ```
+ *
+ * @see {@link https://learn.microsoft.com/en-us/entra/identity-platform/v2-protocols-oidc}
+ */
+export function createAzureADOAuthConfig(config: {
```

Comment:
```suggestion
export function createAzureAdOAuthConfig(config: {
```
Nit: the Deno naming convention doesn't capitalize abbreviations.

Reply:
fixed